src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotConstantPool.java

```diff
@@ -653,7 +653,9 @@ public List<BootstrapMethodInvocation> lookupBootstrapMethodInvocations(boolean
                             .toList();
                     .toList();
         } else {
-            return IntStream.range(1, length()).filter(cpi -> {
+            return IntStream.range(1, length())
+                            .filter(this::isDynamicEntry)
```

Comment:
Looks like you forgot to add the definition of `isDynamicEntry` that I suggested:
```
    private boolean isDynamicEntry(int cpi) {
        JvmConstant tagAt = getTagAt(cpi);
        return tagAt != null && tagAt.name.equals("Dynamic");
    }
```